src/breaking-change.ts

```diff
@@ -115,19 +115,39 @@ export async function runScript() {
     }
   }
 
+  /**
+   * prepare for switch to master branch, 
+   * if not the switching to master below would failed
+   */
+  if (cli.defaultConfig().env.SYSTEM_PULLREQUEST_TARGETBRANCH !== "master") {
+    utils.setUpstreamBranch("master", "remotes/origin/master")
+  }
+
   // create Azure DevOps PR properties.
   const pr = await devOps.createPullRequestProperties(cli.defaultConfig())
 
   // See whether script is in Travis CI context
   console.log(`isRunningInTravisCI: ${isRunningInTravisCI}`);
 
+  console.log(`PR target branch is ${pr ? pr.targetBranch : ""}`)
+
   let targetBranch = utils.getTargetBranch();
   let swaggersToProcess = await utils.getFilesChangedInPR(pr);
 
   console.log('Processing swaggers:');
   console.log(swaggersToProcess);
 
   console.log('Finding new swaggers...')
+
+  /*
+ * always compare against master
+ * we still use the changed file got from the PR, because the master branch may quite different with the PR target branch
+ */
+  if (pr && pr.targetBranch !== "master") {
+    (pr.targetBranch as any) = "master"
```

Comment:
Why as any. I think the targetBranch is string type

Reply:
yes, fixed it